libbeat/docs/shared-ilm.asciidoc

```diff
@@ -79,8 +79,10 @@ For more information, see
 {ref}/indices-rollover-index.html#_using_date_math_with_the_rollover_api[Using
 date math with the rollover API].
 
-NOTE: If you modify this setting after loading the index template, you must
-overwrite the template to apply the changes.
+NOTE: Before modifying this setting for an existing ILM setup, you must manually
+remove any aliases related to the previous pattern, then overwrite the policy.
+Existing indices that don't match the new pattern will no longer be subject to
```

Comment:
s/will/might/

size based policy will not be able to make progress, as no new events are indexed. Time based indices might still be rolled over by ILM.

I guess it depends on the pattern.

Another warning: The code creates the first index with the alias, which most likely will end with `000001`. I'm not sure what happens if another index with the same name already exists (because the user switch the pattern back to an older one).